.eslintrc.js

```diff
@@ -134,7 +134,13 @@ module.exports = {
     yoda: 'error',
 
     // Imports
-    'import/extensions': ['error', 'never'],
+    'import/extensions': [
+      'error',
+      'never',
+      {
+        schema: 'always',
```

Comment:
This exception is specific to [the sourcegraph repo](https://github.com/sourcegraph/sourcegraph). What do you think about adding it to the project's eslint config?